extras/nginx/ideascube

```diff
@@ -14,6 +21,8 @@ server {
     server_name ideasbox.lan www.ideasbox.lan ideasbox.local www.ideasbox.local koombook.lan www.koombook.lan koombook.local www.koombook.local bibliotecamovil.lan www.bibliotecamovil.lan;
     charset     utf-8;
 
+    
+
```

Comment:
it's noise, I first expected to put configuration there and forgot to remove them.